compiler/src/dmd/apply.d

```diff
@@ -11,13 +11,13 @@
 
 module dmd.apply;
 
-import dmd.arraytypes;
-import dmd.dsymbol;
-import dmd.dsymbolsem;
-import dmd.dtemplate;
-import dmd.expression;
-import dmd.root.array;
-import dmd.visitor;
+// import dmd.arraytypes;
```

Comment:
For completely unused imports, if you could split them out into a different PR that would be much appreciated. That PR will be likely to be accepted much faster, will be much smaller.